initial/src/main/java/com/example/resources/CourseResource.java

```diff
@@ -0,0 +1,35 @@
+package com.example.resources;
+
+import com.example.services.CourseService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CourseResource {
+
+    public static final String URL_PREFIX = "/course";
+    public final CourseService _courseService;
+
+    @Autowired
+    /**
+     * @return CourseService object
+     */
+    public CourseResource(CourseService courseService) {
+        _courseService = courseService;
+    }
+
+    @RequestMapping(URL_PREFIX + "/{id}")
+    /**
+     * Returns a string of
+     * @param id The id of the course
+     * @return the course json
```

Comment:
"A Json String that represents a Course object". Feel free to take a look at how to link classes in the javadoc.